addons/package_hierarchy/views/stock_quant_views.xml

```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="quant_package_search_view" model="ir.ui.view">
+        <field name="name">package.search</field>
+        <field name="inherit_id" ref="stock.quant_package_search_view"/>
+        <field name="model">stock.quant.package</field>
```

Comment:
They have both stock.quant and stock.quant.package in the same file, but we like to split it, since they are different objects, makes things easier to find.